ports/geckolib/tools/regen_bindings.sh

```diff
@@ -14,11 +14,19 @@ if [ ! -d rust-bindgen ]; then
   exit 1
 fi
 
-export RUST_BACKTRACE=1
-export LIBCLANG_PATH="$(pwd)/llvm/build/Release+Asserts/lib"
-export DYLD_LIBRARY_PATH="$(pwd)/llvm/build/Release+Asserts/lib"
-export LD_LIBRARY_PATH="$(pwd)/llvm/build/Release+Asserts/lib"
-export DIST_INCLUDE="$1/dist/include"
+# Check for /usr/include
+if [ ! -d /usr/include ]; then
+  echo "/usr/include doesn't exist. Mac users may need to run xcode-select --install."
+  exit 1
+fi
+
+if [ "$(uname)" == "Linux" ]; then
+  PLATFORM_DEPENDENT_DEFINES+="-DOS_LINUX";
+  LIBCLANG_PATH=/usr/lib/llvm-3.8/lib;
```

Comment:
This might be not true for every distro. That said I personally don't need it, so it looks sensible to keep like this and add more thorough tests (a list of common dirs?) if needed in the future.